src/Services/Qrda/Helpers/Cat1View.php

```diff
@@ -123,11 +123,11 @@ public function dose_quantity_value(Mustache_Context $context)
 
     public function result_value(Mustache_Context $context)
     {
+        // TODO Do patient models ever return a string for results?
```

Comment:
@kchapple Still need to verify various patient models QDM result types for observations and labs. You may want to change the way I did helper but this is close.